### PR TITLE
Deprecated concepts

### PR DESCRIPTION
**Pull request by tosan88**
Added possibility to include deprecated concepts based on the boolean flag `include_deprecated` + added isDeprecated field on response (omitted if false).

Also the changes include:
* fixed dredd integration
* moved to dep from govendor
* updated golang version
* adjusted tests

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/Financial-Times/api-endpoint"
+  version = "0.0.2"
+
+[[constraint]]
+  name = "github.com/Financial-Times/go-fthealth"
```
Comment (andreic92) [inline Gopkg.toml]: Is here possible to use [0.5.0](https://github.com/Financial-Times/go-fthealth/releases/tag/0.5.0)?

File: Gopkg.toml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+[[constraint]]
+  name = "github.com/jawher/mow.cli"
+  version = "1.0.3"
+
+[[constraint]]
+  name = "github.com/satori/go.uuid"
+  version = "1.1.0"
+
+[[constraint]]
+  name = "github.com/sirupsen/logrus"
```
Comment (andreic92) [inline Gopkg.toml]: Could you update this to [go-logger](https://github.com/Financial-Times/go-logger/releases/tag/1.0.0)? Or at least to [1.0.5](https://github.com/Financial-Times/go-logger/blob/febee6537e90971bab6f6fe60b71b4a0562dcab3/vendor/vendor.json#L22) for logrus in order to match the version for logrus from go-logger?

Comment (tosan88) [reply Gopkg.toml]: Done

File: Gopkg.toml (hunk 1 above)
Comment (tosan88) [reply Gopkg.toml]: Done

**Pull request merged**